content/blog/10-reasons-why-you-should-be-using-surveys.md

```diff
@@ -8,6 +8,11 @@ author: Mriganka Chandra
 createdAt: 2023-01-20
 coverImg: /img/10-reasons-why-you-are-still-not-doing-surveys.png
 coverImgAlt: Ten reasons why you are still not doing surveys
+images: [
+    https://formester.com/img/10-reasons-why-you-are-still-not-doing-surveys.png,
+    https://formester.com/blog/10-reasons-why-you-should-be-using-surveys/a-quick-way-to-know-what-trending-in-the-market.png",
+    https://formester.com/blog/10-reasons-why-you-should-be-using-surveys/target-the-right-audience-and-make-productive-changes.png
+  ]
```

Comment:
How this will get updated in future blogs ?

Reply:
I think this can be achieve through dynamic image handling but it would require a CMS that will support it.

Reply:
This could be an option in the meta section where the user can select any number of images. Just like they do right now for the hero image.

If that's takes care we should be good.

As we already have the netlify cms integrated.

Reply:
Yes @ankitsinghaniyaz , taking your suggestion, I have added 'selectedImages' in the Netlify CMS configuration that we have. So now, we will have to add images in the 'selectedImages' array for each article, and they will be automatically picked up by the CMS.

<img width="591" alt="Screenshot 2023-05-23 at 11 35 20 AM" src="https://github.com/formester/formester-site/assets/132327557/848071ca-f228-4ee8-bf39-5d3221efce7d"
<img width="931" alt="Screenshot 2023-05-23 at 11 37 36 AM" src="https://github.com/formester/formester-site/assets/132327557/5822479b-32d1-4792-ba00-2a73bcf644c5">
>